### PR TITLE
Feat: Alert/avatar components

### PR DESCRIPTION
**Pull request by AtulHande03a**
Added Alert and Avatar components. Also it is mobile responsive.

Screenshot (35)
Screenshot (36)
Screenshot (37)
Screenshot (38)

### CODE REVIEW

**Review by TusharJade — COMMENTED** [review]
Looks good to me👍

**Review by abhijeetekad — APPROVED** [review]

File: documentation/docs.html [hunk 1]
```diff
@@ -77,8 +77,198 @@
...
+              <iframe
+                loading="lazy"
+                src="https://carbon.now.sh/embed/tPGxVJfjmP9wF4cd39jv"
+                style="
+									height: 200px;
+									border: 0;
+									transform: scale(1);
+									overflow: hidden;"
+                sandbox="allow-scripts allow-same-origin"
+              >
```
Comment (amarphule) [inline documentation/docs.html:105]: Need proper Indentation.

**Review by amarphule — COMMENTED** [review]
Great work, just one suggestion given in the comment.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+.error {
+  color: var(--red);
+  background-color: var(--light-red);
+}
+
+.warning {
+  color: var(--dark-yellow);
+  background-color: var(--light-yellow);
+}
```
Comment (horsemaker) [inline components/alert/alert.css:35]: These 3 seem to be generic classes. You can or you might be using them again somewhere. If that happens do shift them to the main file or the utilities.

**Review by horsemaker — APPROVED** [review]
Looks great! 👌
PhirHeraPheriEkdumKadakMaalHaiGIF

File: components/alert/alert.css (hunk 2 above)
Comment (AtulHande03a) [reply components/alert/alert.css:35]: No those are there only for alert.css, I am not using them anywhere else

File: documentation/docs.html (hunk 1 above)
Comment (AtulHande03a) [reply documentation/docs.html:105]: In the vs code it looks okay but here the indentation is messed up

**Review by anandkumarcoder — APPROVED** [review]
looks good. remember to add importing comment in gearUI CSS file in next branch push

**Review by poojavishnoi — APPROVED** [review]
Looks greatt!

**Pull request merged**